aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -251,6 +251,22 @@ module std::features {
         is_enabled(AGGREGATOR_SNAPSHOTS)
     }
 
+    /// Whether enable MOVE functions to call create_auid method to create AUIDs.
+    /// Lifetime: transient
+    const CONCURRENT_TOKEN_V2_ENABLED: u64 = 31;
+
+    public fun concurrent_token_v2_enabled(): bool acquires Features {
```

Comment:
I wonder if we can just use a single flag for both? They likely would be enabled in the same proposal anyway

Reply:
I thought of having more flexibility with both, but we can combine as well.